### DISCUSSION TITLE
does this work on windows 7?

### DISCUSSION THREAD

**Discussion by ghost** [GrandOrgue/grandorgue#1459] [Help]
basically what's on the title. it says "windows" but doesn't specify which versions exactly.

**Comment by rousseldenis** [comment]
> basically what's on the title. it says "windows" but doesn't specify which versions exactly.

@cyberhardt Yes. Have you tried to install it yet ?

**Reply by ghost** [reply]
I'm not on my windows 7 machine at this moment, so I haven't

**Comment by midimusic** [comment]
It will probably work on any 64 bit version of windows.
csw900
